xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -535,7 +537,7 @@
         <ReturnType>System.String</ReturnType>
```

Comment:
Since this class implements IVSAEngine, can you just use the same descriptions that are used in https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.Vsa/IVsaEngine.xml#L325. I've been copying and pasting so far, but I think it will be faster if you just review the others for me.

Reply:
I've committed your earlier suggestions and will proceed with this next.